Source/WebKit/Configurations/BaseExtension.xcconfig

```diff
@@ -29,6 +29,11 @@ CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING = NO;
 INSTALL_PATH = $(INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 INSTALL_PATH[sdk=*simulator] = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 INSTALL_PATH[sdk=iphone*17.4*] = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
+SKIP_INSTALL = NO;
```

Comment:
Would it be more appropriate to do the opposite, skipping by default, and unskipping on platforms where it's enabled?

Reply:
Ah, good point, will fix!

Thanks for reviewing, all!